benchmarks/src/utils.rs

```diff
@@ -144,7 +142,7 @@ impl BenchmarkCli {
         build_bench(manifest_dir, profile)
     }
 
-    pub fn bench_from_exe<VC>(
+    pub fn bench_from_exe<VC, E: StarkFriEngine<SC>>(
```

Comment:
should/can we keep this function for fixed engine just for convenience? so we don't need to change all the benchmarks?